orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -50,7 +57,9 @@ def __init__(self):
         self.output_mask = []           # Output corpus indices
         self.doc_webview = None         # WebView for showing content
         self.search_features = []       # two copies are needed since Display allows drag & drop
-        self.display_features = []
+
+        self.search_indices = []
```

Comment:
Is this initialization to empty arrays still needed? If so, how that it's not needed for `display_features `?